docker_mcp/models/params.py

```
@@ -0,0 +1,195 @@
+    )
+    ssh_user: str = Field(
+        default="", 
+        description="SSH username"
+    )
+    ssh_port: int = Field(
+        default=22, 
+        ge=1, 
+        le=65535, 
+        description="SSH port number"
+    )
+    ssh_key_path: Optional[str] = Field(
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)